source/core/src/main/com/csse3200/game/areas/terrain/TerrainCropTileFactory.java

```diff
@@ -56,8 +56,6 @@ public static Entity createTerrainEntity(Vector2 position) {
 
 		tile.setPosition(position);
 		logger.debug("Registering crop tile {} with entity service", tile);
```

Comment:
remove the debug as well

Reply:
Did you want the first debug removed aswell??? 

`logger.debug("Creating crop tile at position {}", position);`